azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/jobpage.vm

```diff
@@ -21,7 +21,7 @@
   #parse ("azkaban/webapp/servlet/velocity/style.vm")
   #parse ("azkaban/webapp/servlet/velocity/javascript.vm")
 
-  <script type="text/javascript" src="${context}/js/azkaban/view/job-edit.js"></script>
+  <script type="text/javascript" src="${context}/js/azkaban/view/job-edit.js?v=1597214414"></script>
```

Comment:
What does this change do? Is "1597214414" a well known number in velocity?

Reply:
It is the timestamp. This timestamp guarantees that a page older than it will reload using newer code.

Reply:
Deepak is correct. That's a trick to change the url of the js file and therefore force its reload for all clients.

Reply:
Thanks for the explanation.